lib/classes/tile.js

```diff
@@ -86,7 +87,8 @@ GridGame.classes.tile = function (x, y) {
     .removeClass(function(index, css) {
       return (css.match (/\btile_\S+/g) || []).join(' ');
     })
-    .toggleClass('tile_city', this.city);
+    .toggleClass('tile_city', this.city)
+      .toggleClass('tile_rock', this.rock);
```

Comment:
Fair point. It looks like everything here was a level of indents too low, and the new line was put to the right level by my IDE.